robottelo/constants.py

```diff
@@ -861,7 +861,8 @@
     {'name': 'SmartProxy', 'controller': 'smart_proxies', 'skip_for_ui': True},
     {'name': 'Compute_Resource', 'controller': 'compute_resources',
         'setup': entities.DockerComputeResource},
-    {'name': 'Compute_Profile', 'controller': 'compute_profiles'},
+    {'name': 'Compute_Profile', 'controller': 'compute_profiles',
+        'skip_for_ui': True},
```

Comment:
@rplevka I also see that Compute profile page is empty by default, so search button is not present.  Is this a bug?  I am not sure if it is a bug so I skipped this too for now. Please let me know.